CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.11.2
```

Comment:
Thanks for writing a changelog notes and documentation!

Since this is an added feature, our semver bump will be a minor version, up to `2.12.0`. I prefer just just `## HEAD`, though, for the yet-unpublished changes, since other changes might be added to the section, affecting the next version number, before release.